examples/nlp/language_modeling/conf/bert_pretraining_from_preprocessed_config.yaml

```diff
@@ -0,0 +1,73 @@
+# BERT Pretraining from Preprocessed (tokenized) data
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU, or list with gpu indices
+  num_nodes: 1
+  max_epochs: null # the number of training epochs
```

Comment:
ran this with max_steps and got   File"///envs/nemo_env/lib/python3.7/site-packages/pytorch_lightning/trainer/training_loop.py", line 345, in train
    for epoch in range(self.current_epoch, self.max_epochs):
TypeError: 'NoneType' object cannot be interpreted as an integer

Reply:
I'll investigate, it must have started after I rebased because I had been testing that case.

Reply:
Actually, I don't know how it's passing CI because the default config uses max steps.

Reply:
Yeah, error doesn't show up when using +trainer.fast_dev_run=true. So that's how it's passing CI.

Reply:
Okay, so in PTL you have to set max_epochs. If you additionally set max_steps, then it will stop when either max_epoch or max_steps is reached first. By default in PTL, max_epochs is set to 1000, so if we remove max_epochs from the config, then I think we should be okay.

Reply:
Please try it again, but I think this is resolved.